core/src/main/java/com/linecorp/armeria/common/DefaultHttpData.java

```diff
@@ -43,22 +39,15 @@ public DefaultHttpData(byte[] data, int offset, int length, boolean endOfStream)
         return data;
     }
 
-    @Override
-    public int offset() {
-        return offset;
-    }
-
     @Override
     public int length() {
-        return length;
+        return data.length;
     }
 
     @Override
     @SuppressWarnings("ImplicitArrayToString")
     public String toString() {
         return MoreObjects.toStringHelper(this)
-                          .add("offset", offset)
-                          .add("length", length)
                           .add("array", data.toString()).toString();
```

Comment:
This will yield a string like `[B@...`. How about just doing `.add("length", data.length)`?

Reply:
How about removing `.add("array", ...)`?

Reply:
Ah I see, thanks